File: src/uri_beecrowd/Accepted/BEGINNER/P3039_Santa_toys.py
if __name__ == '__main__':
    n = int(input())
    M = F = 0
    for x in range(0, n):
        name, sex = map(str, input().split(' '))
        if sex == 'M':
            M += 1
        else:
            F += 1
    print(M, 'carrinhos')
    print(F, 'bonecas')

File: Java/URI_BEECROWD/BEGINNER/P3039_Santa__toys.java
package URI.Accepted.BEGINNER.sourcecode;


import java.io.IOException;
import java.io.BufferedReader;
import java.io.InputStreamReader;

public class P3039_Santa__toys {

    static int M = 0, F = 0;

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        for (int i = 0; i < n; i++) {
            String[] st = br.readLine().split(" ");
            String t = st[1];
            if (t.equals("M")) {
                M++;
            } else {
                F++;
            }
        }
        System.out.println(M + " carrinhos");
        System.out.println(F + " bonecas");
    }
}
